### DISCUSSION TITLE
Small 'Unlock Database' Popup

### DISCUSSION THREAD

**Discussion by flameshikari** [keepassxreboot/keepassxc#10183] [Q&A] [answered]
Is there a way to call KeePassXC for database unlock on Windows and have that small popup?

![2024-01-12_17-48-55_719](https://github.com/keepassxreboot/keepassxc/assets/17804422/15856fbd-2fcf-4b0b-adec-af839c90ed92)

A popup like this shows only when clicking on `Reopen database` in KeePassXC-Browser.

**Comment by droidmonkey** [comment, marked as answer]
We show that popup when using the browser extension or Auto-Type. Otherwise you have to focus the keepassxc window and unlock the database. There is a request for a global shortcut to bring the keepassxc window to focus.

**Reply by flameshikari** [reply]
Thanks for the answer! However, is there no way to call that popup with some command or utility? I wanted to call it through a script. Sorry for a similar question, just added more clarification.

**Reply by droidmonkey** [reply]
No
